packages/rich-text/src/plugins/DeserializeDocx/cleanHtmlEmptyElements.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ The cleanHtmlEmptyElements from Plate is overly aggressive in removing elements, removing those that are necessary for maintaining proper spacing (span).
```

Comment:
Is there an open upstream issue we can link here? are maintainers aware of the behavior?

Reply:
Didn't find one, I'll create one 👍